src/AspNet.Security.OAuth.Gitter/GitterAuthenticationExtensions.cs

```diff
@@ -4,10 +4,10 @@
  * for more information concerning the license and the contributors participating to this project.
  */
 
-using System;
 using AspNet.Security.OAuth.Gitter;
-using JetBrains.Annotations;
-using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+using System;
```

Comment:
`System` using always go first.

Reply:
Interesting. Visual Studio 2017 puts them in alphabetically order.